src/XtensaOptimize.cpp

```diff
@@ -937,8 +938,11 @@ class MatchXtensaPatterns : public IRGraphMutator {
             {"halide_xtensa_narrow_i48_with_rounding_shift_i16", i16(rounding_shift_right(i32(wild_i48x), wild_i32))},
             {"halide_xtensa_narrow_i48_with_rounding_shift_u16", u16(rounding_shift_right(u32(wild_i48x), wild_u32))},
 
+#if 0  // Somehow this pattern is preventing a i16(i32(i48x) >> wild_i32) to be recognized as a PACKVRNRNX48
```

Comment:
Do we need to open an Issue for this?

Reply:
I think I also ran into this issue recently and have a fix.

Reply:
This needs to be resolved before submission, I'll double check if it's the same issue I had.

Reply:
Resolved now, but you'll need to sync.